iabtcf-encoder/src/test/java/com/iabtcf/encoder/BitWriterTest.java

```diff
@@ -336,7 +334,7 @@ public void testSimpleEncode() {
         BitWriter bw = new BitWriter();
         bw.write(2, 6);
         bw.write(Instant.parse("2020-01-26T17:01:00Z").toEpochMilli() / 100, 36);
-        bw.writeDays(Instant.parse("2021-02-02T17:01:00Z"), FieldDefs.CORE_CREATED);
+        bw.write(Instant.parse("2021-02-02T17:01:00Z"), FieldDefs.CORE_LAST_UPDATED);
```

Comment:
hmm.. why was this changed from CORE_CREATED to LAST_UPDATED field?

Reply:
I think this is correct..

        bw.write(2, 6);  => version field
        bw.write(Instant.parse("2020-01-26T17:01:00Z").toEpochMilli() / 100, 36); => created field
        bw.write(Instant.parse("2021-02-02T17:01:00Z"), FieldDefs.CORE_LAST_UPDATED); => updated field.